x-pack/plugins/session_view/public/components/process_tree/hooks.ts

```diff
@@ -134,6 +134,14 @@ export class ProcessImpl implements Process {
     return '';
   }
 
+  getEndTime() {
+    const endEvent = this.filterEventsByAction(this.events, EventAction.end);
```

Comment:
I believe it's better to use `findEventsByAction` instead, i.e

```js
getEndTime() {
	 const endEvent = this.findEventsByAction(this.events, EventAction.end);
   return endEvent?.['@timestamp'] || ''
}
```

Reply:
@zizhouW  mentioned to me to get the last one in the case we have more than 1 same event

Reply:
Yea I thought it would be safer this way since we stopped assuming that we will be getting correct data.

Reply:
oh, good point

Reply:
There should never be two 'end' events for a single process.